planning_domains_api.py

```diff
@@ -116,7 +143,7 @@ def get_tags():
 
 def get_collections(ipc = None):
     """Return the collections, optionally which are IPC or non-IPC"""
-    res = query('/classical/collections/')
+    res = query('/collections/')
```

Comment:
Not going to leave comments on them all, but I think they should have a formalism parameter and pass it on to query to include in the URL.